### ISSUE TITLE
Patcher script breaks mono spacing for Input font

### ISSUE THREAD

**Issue by MrVizious** [ryanoasis/nerd-fonts#564]
#### 🎯 Subject of the issue

I tried to patch the Input Mono font, but after doing so, the spacing is weird, as if it was Sans or similar.
Used the command:
```
fontforge -script ./font-patcher --careful --complete  --progressbars --mono /path/to/font
```
(substituting the /path/to/font with my own, ofc)

After patching it, the mono spacing seems to be wiped off and horizontal spacing is all messed up.

#### 🔧 Your Setup

- _Which font are you using?_
`Input Mono`
- _Which terminal emulator are you using?_
`Alacritty`
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_
`Linux Manjaro`

#### ★ Optional
Regular Input Mono:
![image](https://user-images.githubusercontent.com/38861496/104855918-cc2fd380-590f-11eb-8fd9-b3369350090c.png)

Patched Input font:
![image](https://user-images.githubusercontent.com/38861496/104856005-4a8c7580-5910-11eb-908e-c09bde2b548a.png)

**Comment by qwattash** [comment]
I have been having a similar issue, check that `fc-match 'XXX Nerd Font`:lang=en:weight=regular` actually returns the font you are expecting. I was having the issue because the font was not matched properly.

**Comment by Finii** [comment]
Fixed with 3.0.0
